script/tag.sh

```diff
@@ -0,0 +1,56 @@
+# Update and commit linebot/version.go and create a new tag.
+#
+# Usage:
+#   bash script/tag.sh v3.0.0
+
+if [ $# -gt 1 ]; then
+  echo "too many arguments" > /dev/stderr
+  echo 'Usage: tag.sh $TAG' > /dev/stderr
+  exit 1
+fi
+
+if [ $# -lt 1 ]; then
+  echo "TAG argument is required" > /dev/stderr
+  echo 'Usage: tag.sh $TAG' > /dev/stderr
+  exit 1
+fi
+
+TAG=$1
+echo "TAG: $TAG"
+VERSION=${TAG#v}
+
+if [ "$TAG" = "$VERSION" ]; then
+  echo "TAG must start with 'v'"
+  exit 1
+fi
+
+echo "cd `dirname $0`/.."
+cd `dirname $0`/..
+
+echo "create linebot/version.go"
+cat << EOS > linebot/version.go
+// Copyright 2016 LINE Corporation
```

Comment:
2018?

Reply:
Oh, sorry..

Reply:
Oh, I misunderstood.
This 2018 is correct. linebot/version.go has created before 2018.
I should add LICENSE at the head of tag.sh.

Reply:
I have added the license.
https://github.com/line/line-bot-sdk-go/pull/103/commits/87cba7bae8f23c6f8a887b2dbfd1f2f2b8878cc0

Reply:
ah, I was wrong, thank you 🙇